### ISSUE TITLE
config rule s3 Lambda - no resources matched

### ISSUE THREAD

**Issue by sudharsans** [capitalone/cloud-custodian#1451]
I have custodian policy which doesn't work from lambada but works on the local machine. following is the logs found in the cloudwatch. Could you please help?

```
 - name: S3-enable-logging
   mode:
     type: config-rule
     role: arn:aws:iam::account:role/CloudCustodian
   resource: s3
   filters:
      - "tag:Name": present
   actions:
     - type: toggle-logging
       target_bucket:s3-logs
       target_prefix: logs
```

Policy:
```
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Effect": "Allow",
      "Action": "*",
      "Resource": "*"
    }
  ]
}
```

Logs:
> Filtering resources with [<c7n.filters.core.ValueFilter object at 0x7f0303204ad0>]
> applied filter <c7n.filters.core.ValueFilter object at 0x7f0303204ad0> 1->0
> Filtered from 1 to 0 s3
> [INFO] Filtered resources 0
> [INFO] policy: S3 resources: s3 no resources matched
> [INFO] policy: S3 resources: s3 no resources matched

**Comment by sudharsans** [comment]
I have full admin access in the policy.

**Comment by sudharsans** [comment]
@kapilt updated the question with custodian policy and iam role

**Comment by kapilt** [comment]
the underlying issue is that config has some mangling of a resource thats a little distinct then a straight describe call. the branch here normalizes the config representation to match that of the describe, its fallen into the backlog at the moment though https://github.com/capitalone/cloud-custodian/pull/907

**Comment by kapilt** [comment]
fixed with merge of #907
